.vim/sky8336/plugin.vim

```diff
@@ -89,11 +89,11 @@ let plugin_enalbe_Qt_highlight_support = 1
 		Plug 'octol/vim-cpp-enhanced-highlight'
 	endif
 
-	if plugin_enalbe_rainbow_parentheses == 1
+	if plugin_enable_rainbow_parentheses == 1
```

Comment:
modify the spelling error : 
1.
36  let plugin_enalbe_rainbow_parentheses = 0       
...

2.
plugin_setting.vim:
681 if plugin_enalbe_rainbow_parentheses == 1